sunpy/net/vso/attrs.py

```diff
@@ -20,6 +20,7 @@
 from datetime import datetime
 
 from astropy import units as u
+from astropy.time import Time as astropyTime
```

Comment:
why not just import it as `Time`?

Reply:
Because it collides with sunpy's time.
Same name.

Reply:
Ooohh, I see, vso's `Time`.